lib/ui/painting.dart

```diff
@@ -861,9 +861,27 @@ class MaskFilter extends NativeFieldWrapperClass2 {
   ///
   /// A blur is an expensive operation and should therefore be used sparingly.
   MaskFilter.blur(BlurStyle style, double sigma) {
-    _constructor(style.index, sigma);
+    _constructorBlur(style.index, sigma);
   }
-  void _constructor(int style, double sigma) native "MaskFilter_constructor";
+  void _constructorBlur(int style, double sigma) native "MaskFilter_constructorBlur";
+
+  MaskFilter.shadow(double occluderHeight,
+                    double lightPosX,
+                    double lightPosY,
+                    double lightPosZ,
+                    double lightRadius,
+                    double ambientAlpha,
+                    double spotAlpha) {
```

Comment:
We should make these named arguments.  That will make the call sites much more readable.